images/test/busybox/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM busybox
```

Comment:
Is this file really needed? It is not particularly hard to get a busybox image and there is nothing special about the image (any other image that is not a toolbox image would be sufficient).

Reply:
I added this file to build the hosted image in the quay.io repository from it. When I created the quay.io image I just uploaded this Dockerfile. We can remove it from our code but I considered to add it to "subliminally" add the information about what image are we using at quay.io and where it comes from.

I know any other image could do the work, but as busybox is the smaller we can find out there, I think busybox is ok for what we need.

Reply:
> I know any other image could do the work, but as busybox is the smaller we can find out there, I think busybox is ok for what we need.

Yes, that is undoubtedly true.

> I added this file to build the hosted image in the quay.io repository from it.

Fair enough. Maybe just add a `README.md` explaining the purpose of the Dockerfile.

One more thing. Maybe rename the file to `Containerfile`. The name is a bit more "generic".